src/components/FeaturedEventCard.tsx

```diff
@@ -95,20 +109,56 @@ const FeaturedEventCard: React.FC<FeaturedEventProp> = ({
 					<ArrowDownUp rotate={panelOpen} />
 				</StyledIconText>
 				<StyledCourseInfo open={panelOpen}>
-					<h4>About the Course</h4>
-					<StyledText>{event.courseInfo.course}</StyledText>
-					<h4>About the Instructor(s)</h4>
-					<StyledList>
-						{event.courseInfo.instructor.map((info, index) => (
-							<li key={index}>{info}</li>
-						))}
-					</StyledList>
-					<p>Refreshment: {event.refreshment === true ? 'Yes' : 'No'}</p>
-					<p>Starter Pack: {event.starterPack === true ? 'Yes' : 'No'}</p>
-					<StyledSpanLink>
-						Please note our{' '}
-						<Link href="/">COVID-19 Protocol & social distancing measures</Link>
-					</StyledSpanLink>
+					<CourseInfoFlex>
+						<CourseInfoDiv>
+							<h4>Description</h4>
+							<StyledText>{training.description}</StyledText>
+							{training.prerequisites && (
+								<>
+									<h4>Prerequisites</h4>
+									<StyledText>{training.prerequisites}</StyledText>
+								</>
+							)}
+							<h4>Benefits</h4>
+							<StyledList>
+								{training.benefits.map((info, index) => (
+									<li key={index}>{info}</li>
+								))}
+							</StyledList>
+							{training.extraInformation && (
+								<Text>Note: {training.extraInformation}</Text>
+							)}
+							<Text>
+								Refreshment: {training.refreshment === true ? 'Yes' : 'No'}
+							</Text>
+							<Text>
+								Starter Pack: {training.starterPack === true ? 'Yes' : 'No'}
+							</Text>
+							<MoreInfoBox>
+								To attend this training, please register at least two working
+								days before the event.{' '}
+								{training.nextTrainingDate && (
+									<span>
+										{`The next training will take place on ${formatDate(
+											new Date(training.nextTrainingDate),
+										)}`}
+									</span>
+								)}
+							</MoreInfoBox>
+						</CourseInfoDiv>
+
+						{training.eventPosterImage && (
+							<CourseImageDiv>
+								<Image
+									// src={`http:${training.eventPosterImage}`}
+									src={`https://res.cloudinary.com/dcfntkzap/image/upload/v1690004091/Training-data-images/Event_Flier_Template_Single_Person_yaj9is.png`}
```

Comment:
this link, you'll pass it through dynamically the cms, right?

Reply:
Yeah, the demo data doesn't have a valid image url and leads to an error. That's why i hard coded it.
when there is valid url in the dummy Data or live Data, i can revert it.